### PR TITLE
007 redirects

### PR DESCRIPTION
**Pull request by Guo-William**
#7 
Redirect to log-in page after logging out from any page
Prevent users from viewing any page other than the log-in page if they are not logged in
Prevent forced log-outs on page refreshes

Does not do:
Add user's username to profile page path

Will be doing that in #14 instead as it makes more sense to do there.

### CODE REVIEW

File: travelpal/assets/js/components/LoginForm.jsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ function LoginForm(props) {
     <React.Fragment>
       <div className="header">
         {/* Image source: https://www.shareicon.net/transport-flight-aeroplane-airplane-airport-transportation-travel-plane-824400 */}
-        <img src="images/airplane-icon.png" alt="logo" />
+        <img src="/images/airplane-icon.png" alt="logo" />
```
Comment (kimberlypn) [inline travelpal/assets/js/components/LoginForm.jsx]: Need to make this change in `Nav.jsx` as well. After logging out and then logging in, the logo didn't load for me until after I added the `/` in `Nav.jsx`.

Not sure why the logo shows up the first time I log in before making this change, though.

Comment (Guo-William) [reply travelpal/assets/js/components/LoginForm.jsx]: I takes part of the current url and adds it to the img path, thats why

Comment (Guo-William) [reply travelpal/assets/js/components/LoginForm.jsx]: https://github.com/kimberlypn/TravelPal/pull/20/commits/700aa97d0478caa7ee087f41897391662adcfd3f

**Comment by Guo-William** [comment]
@kimberlypn this should fix the registration form https://github.com/kimberlypn/TravelPal/pull/20/commits/d2b80e105a3ff2d4e1ac9d6c738165390cce81ae

Note I also hide the registration form once they successfully register so the login is a bit more smooth.

**Comment by kimberlypn** [comment]
Gasp. You didn't squash and merge.

**Comment by Guo-William** [comment]
My bad will do that from now on

**Pull request merged**